crates/rattler_conda_types/src/channel/mod.rs

```diff
@@ -449,6 +448,17 @@ fn absolute_path(path: &str, root_dir: &Path) -> Result<Utf8TypedPathBuf, ParseC
         return Ok(path.normalize());
     }
 
+    // Parse the `~` as the home folder
+    if path.starts_with("~/") {
+        return Ok(Utf8TypedPathBuf::from(
+            dirs::home_dir()
+                .ok_or(ParseChannelError::InvalidPath(path.to_string()))?
+                .to_string_lossy()
+                .as_ref(),
+        )
+        .join(path.strip_prefix("~/").unwrap()));
+    }
```

Comment:
You can avoid an `unwrap` and also account for windows paths:

```suggestion
    if let Some(user_path) = path.strip_prefix("~/").ok().or_else(|| path.strip_prefix("~\\").ok()) {
        return Ok(Utf8TypedPathBuf::from(
            dirs::home_dir()
                .ok_or(ParseChannelError::InvalidPath(path.to_string()))?
                .to_string_lossy()
                .as_ref(),
        )
        .join(user_path);
    }
```